.github/workflows/ci.yml

```diff
@@ -87,3 +87,30 @@ jobs:
         with:
           command: tarpaulin
           args: --coveralls $TOKEN --avoid-cfg-tarpaulin # @TODO restore to normal (https://github.com/xd009642/tarpaulin/issues/756#issuecomment-838769320)
+
+  dockerhub:
+    name: Docker build and push to Docker Hub
+    runs-on: ubuntu-latest
+
```

Comment:
Could you remove this line? It does not help to understand the data-structure

Reply:
Do you mean these `name:` lines?